### ISSUE TITLE
image scaling not reflected in layer thumbnail

### ISSUE THREAD

**Issue by alisterburt** [napari/napari#5684]
## 🐛 Bug

The thumbnail rendering of an image layer shows the data pre-scaling
<img width="1624" alt="image" src="https://user-images.githubusercontent.com/7307488/228909661-a57a167c-6301-4f51-92b0-15ef437ab332.png">

## To Reproduce

```python
import numpy as np
import napari

image = np.random.random(size=(100, 400))
viewer = napari.Viewer()
viewer.add_image(image, scale=(4, 1))
napari.run()
```

## Expected behavior

scaled image should be shown in layer thumbnail

**Comment by jni** [comment]
Good idea! I think we just need to divide the `ndi.zoom` scale factors by the scale... But of course there'll be some axis order bugs lurking... 😬
